vignettes/including-data-in-teal-applications.Rmd

```diff
@@ -161,10 +175,10 @@ The filter panel supports MAEs out of the box.
 
 ##### `datanames`
 
-The `datanames` property lists the objects stored in the `teal_data` that constitute datasets of interest.
-Objects passed to `teal_data` become automatically listed in the `datanames` property of the resulting object.
-Objects created in `teal_data` by evaluating code need not be data objects of interest and as such they are not automatically added to `datanames`.
-Use the `datanames` function to modify the `datanames` property.
+The `datanames` property lists the variables stored in the `teal_data` object that constitute datasets of interest.
+Variables passed to `teal_data` become automatically listed in the `datanames` property of the resulting object.
+Variables created in `teal_data` by evaluating code need not be data objects of interest and as such they are not automatically added to `datanames`.
+`datanames` can be read or modified with the `datanames` function.
```

Comment:
The original wording make more sense to me. Referring to the code example, `iris` and `cars` are dataset objects, they are not "variables". Variables are always referring to the columns within a dataset, `iris$Sepal.Length` is a variable.
Suggesting to use "data objects" or "dataset objects" instead.